en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+
+1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in. This opens the project home page.
+2. If you already have one or more components in your project, click **+ Create**. Otherwise, proceed to the next step.
+3. Click the **API Proxy** card. This opens the **Create an API Proxy** pane.
+
+4. Click on **Authorize With GitHub** card under **Connect your own repository** section to connect your GitHub account.
+    - If you haven’t connected your GitHub repository to Choreo, enter your GitHub credentials and select the repository you forked in the prerequisites section to install the [Choreo GitHub App](https://github.com/marketplace/choreo-apps).
+
+    !!! note
+        The **Choreo GitHub App** requires the following permissions:
+         - Read and write access to code and pull requests.
+         - Read access to issues and metadata.
+        
+        You can [revoke access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-authorized-integrations#reviewing-your-authorized-github-apps) if needed. Write access is only used for sending pull requests; Choreo will not push changes directly to your repository.    
+
+5. Enter the following repository details:
+
+    | **Field**              | **Value**          |
+    |------------------------|--------------------|
+    | **Organization**       | Your GitHub account|
+    | **Repository**         | choreo-samples     |
+    | **Branch**             | **`main`**         |
+    | **API Directory**      | /choreo-samples/department-api-proxy-from-github |
+
+6. Specify the following values as API proxy details:
+
+    !!! info
+        The **Context** field must be unique and cannot be changed after creation.**Version** and **Target** are mandatory fields. **Target** can be changed at any time after the creation.
+
+    | **Field**       | **Value**                                  |
+    |-----------------|--------------------------------------------|
+    | **Context**     | `department-service`                                   |
+    | **Version**     | `v1.0`                                      |
+    | **Target**      | `https://samples.choreoapps.dev/company/hr`|
+
+7. Specify the following values as component details:
+
+    !!! info
+        The **Component Name** field must be unique and cannot be changed after creation. This value is automatically generated, but you can edit it if necessary.
+        **Component Display Name** is a required field.
+
+    | **Field**       | **Value**                                  |
+    |-----------------|--------------------------------------------|
+    | **Component Display Name**| `Department Service`                    |
+    | **Component Name**        | `departmentService`                    |
```

Comment:
Shall we go with `department- service`? There is a validation error when uppercase letters are used.

Reply:
Lets use the automatically suggested name on the UI